setup-machine.sh

```diff
@@ -45,10 +52,15 @@ sudo cp config/20auto-upgrades /etc/apt/apt.conf.d/
 # make sure machine never shuts down on idle, and does shut down on power key (no hibernate or anything.)
 sudo cp config/logind.conf /etc/systemd/
 
+# directory structure
+sudo mkdir /vx
+sudo mkdir /vx/code
+sudo mkdir -p /vx/data/module-scan
```

Comment:
Will this be writable by `vx-*`? I don't see a `chown` for it.

Reply:
in fact no, I just tried ballot-scanner and it failed :) Will fix.

Reply:
fixed!